src/main/java/TalkingJohn/Event.java

```diff
@@ -0,0 +1,33 @@
+package TalkingJohn;
+
+/**
+ * Represents an event task that has a start date and an end date.
+ */
+public class Event extends Task {
+    protected String by1;
```

Comment:
It might be easy to confuse these 2 variables, maybe you could use `from` and `to` or `start` and `end` or something similar?